### PR TITLE
docker container: mount via export

### LINKED ISSUE #205: mount containers without sudo
We can now mount images without superuser privileges, we can also add this to containers.

### PR DESCRIPTION
**Pull request by TomasTomecek**
which means, we no longer need atomic

Fixes #205 

TODO:
* [x] use `docker export` instead

### CODE REVIEW

**Comment by dhodovsk** [comment]
For python2 tests, the execution of docker_container.mount freezes at some point. See also here https://liver3.lab.eng.brq.redhat.com/jenkins/job/MTF-github-1minuteTIP/423/console where it has been aborted.

**Comment by TomasTomecek** [comment]
should be done now

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/DU29](https://da.gd/DU29)__
* Comment: __See details for logs__

File: conu/utils/__init__.py [hunk 1]
```diff
@@ -297,3 +289,58 @@ def graceful_get(d, *args):
...
+    :param client: instance of docker.APIClient
+    :param container: instance of DockerContainer
+    :param path: str, path to a directory, doesn't need to exist
+    :return: None
+    """
+    # we don't do this because of a bug in docker:
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1570828
+    # stream, _ = client.get_archive(container.get_id(), "/")
+    check_docker_command_works()
+    export_p = subprocess.Popen(
```
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply conu/utils/__init__.py]: this doesn't make any sense

[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply conu/utils/__init__.py]: Yes, we can turn off this check...

**Comment by TomasTomecek** [comment]
Addressed one codacy warning, the others are bollocks.

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/AbIb](https://da.gd/AbIb)__
* Comment: __See details for logs__

**Pull request merged**